@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1712,7 +1719,23 @@ export class TextElementsRenderer {
         renderParams: RenderParams
     ): boolean {
         const poiLabel = labelState.element;
-        const worldPosition = poiLabel.position;
+        const worldPosition = this.m_tmpVector3.copy(poiLabel.position);
+
+        const worldOffsetShift = getPropertyValue(
+            (poiLabel.poiInfo?.technique as PoiTechnique)?.worldOffset,
+            this.m_viewState.env
+        );
```

Comment:
PoiInfo properties are set in PoiManager.checkCreateTextElement(). I guess this should go there, and worldOffsetShift should be saved to PoinInfo object.